### ISSUE TITLE
Is there a way to apply LegacyFix on launchers like MultiMC or PrismLauncher, etc.

### ISSUE THREAD

**Issue by AlAndalusWarrior** [betacraftuk/legacyfix#5]

**Comment by notdevcody** [comment]
Yes, you need to download the jar by going to the [Actions](https://github.com/betacraftuk/legacyfix/actions) tab, picking a run and downloading the artifact, then you can specify a JVM argument `-javaagent:path/to/legacyfix.jar`, which will apply LF.
